fe/src/utils/axios.ts

```diff
@@ -53,5 +57,5 @@ instance.interceptors.response.use(
 function refreshToken(instance: AxiosInstance) {
   return instance.post('/auth/token/refresh/', {
     refresh: localStorage.getItem('refreshToken'),
-  })
+  }).catch(() => router.push('/login'))
```

Comment:
I don't think this will work. The loginform does a check (authenticationStore.checkLogin), which will go to /projects whenever a key is present in the localstorage. That key should be removed.

I think `}).catch(authenticationStore.logout)` will fix the issue, since this method also removes the keys from the localstorage.

Reply:
you still need the arrow function but your suggestion might work